stories/index.js

```diff
@@ -129,3 +134,8 @@ storiesOf('Video Plugin')
   .add('Editor with Video Plugin', () => <SimpleVideoEditor />)
   .add('Editor with custom themed Video Plugin', () => <CustomVideoEditor />)
   .add('Editor with Video Plugin and Add Video Button', () => <CustomAddVideoEditor />);
+
+storiesOf('StaticToolbar Plugin')
+  .add('CustomToolbarEditor', () => <CustomToolbarEditor />)
+  .add('Simple toolbar editor', () => <SimpleToolbarEditor />)
+  .add('ThemedToolbarEditor', () => <ThemedToolbarEditor />);
```

Comment:
Mind adding it to the InlineToolbar as well please

Reply:
I'll see what I can do. Is it an option for you to just edit the PR to include this?

Reply:
If I have to I will but if you can help it'd be great since maintaining the repository is already quite a lot of work :heart: